src/assets/wise5/components/common/cRater/CRaterResponses.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * CRaterResponses is the response object from the CRater API.
+ * This is different from CRaterResponse, which is the object that WISE creates to store the
+ * response.
+ */
+export interface CRaterResponses {
+  advisories: string[];
+  feedback: any;
+  response_id: string;
+  scores?: any;
+  trait_scores: any;
+}
```

Comment:
Having this interface name be plural is slightly confusing. Would it be better to name it RawCRaterResponse or CRaterRawResponse or something like that?